src/merchants.md

```diff
@@ -292,6 +292,7 @@ See [Merchant configuration model](#merchant-configuration-model) for expected r
 |:--------------------------------------------|:-------|:-------------------------------------------------------------------------------------|
 | {% break . paymentOptions[\*].walletId %}   | String | Merchant's Centrapay wallet to receive payments. Required for `centrapay.nzd` types. |
 | {% break . paymentOptions[\*].terminalId %} | String | Merchant's Epay terminal id. Required for `epay.nzd` types.                          |
+| {% break . paymentOptions[\*].zapAddress %} | String | Merchant's public zap address. Required for `zap.main` types.                        |
```

Comment:
I think this should be "wavesAddress" and "Merchant's public Waves address".

Reply:
what about `wavesPublicAddress` just to be more precise?